object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
```

Comment:
The items in the action bar below the content are driven purely based on selection, so we can simplify their definition to single/multi select.  If we assume single select, then we only need to specify when multi is allowed